### DISCUSSION TITLE
More helpful boundaries (of nature parks in particular)

### DISCUSSION THREAD

**Discussion by skosch** [organicmaps/organicmaps#977] [Ideas]
National/state/provincial parks, conservation areas, etc. are currently indicated with a diagonal, semi-transparent hatching.

This seemed like a elegant choice to me at first. But on recent hikes, I've actually found it much more distracting than helpful: when I'm zoomed out enough to care about where I am in relationship to the park (for driving/travel planning) I can't see it; when I'm zoomed in enough to find my way while I'm on the mountain, the fact that I'm on park land isn't useful information.

The traditional (and by far the best, IMHO) way to demarcate political boundaries like those of a park is by showing

1. the outline, using a dash-dot line with an inner fringe (as shown on the left here, though I think the green hatched fringe in the second example looks better than the red one)

![image](https://user-images.githubusercontent.com/554383/127784679-19cb06fe-0b99-4f5e-ab27-9921f8d8ed89.png)

2. the region's name, which can either be displayed tracked out across the region (as in "Ruhegebiet Kalkkögel" below) or, at higher zoom levels, along the boundary (as in "Alberta/British Columbia" below):

![image](https://user-images.githubusercontent.com/554383/127788368-e8d1b220-fd86-40d0-bd57-eca4b418ce25.png)

**Comment by biodranik** [comment]
Thanks, it's a good idea. Just need to implement it properly.

**Comment by patepelo** [comment]
This is a great topic to keep discussing. I believe National Parks and ther protected areas are not very well represented on the map.
